decidim-initiatives/app/views/decidim/initiatives/create_initiative/fill_data.html.erb

```diff
@@ -34,7 +34,7 @@
             </div>
 
             <div class="field">
-              <%= f.editor :description, lines: 8, toolbar: :full %>
+              <%= text_editor_for(f, :description, lines: 8, toolbar: :basic ) %>
```

Comment:
Can you leave it with `:full` 🙏🏽??
```suggestion
              <%= text_editor_for(f, :description, lines: 8, toolbar: :full ) %>
```

As far as I see there are some features missing with :basic that they're actually used in production, ie the titles that are missing with this change:

![image](https://user-images.githubusercontent.com/717367/203304491-4cb51ddb-fcc4-4748-a74e-fbb7bb75dbe7.png)


Are used [here](https://www.decidim.barcelona/initiatives/i-1512) in the title "Requisits per signar" (`h5`). 

It should be applied in the other forms too